### DISCUSSION TITLE
ghelper doesnt detect my peripherals

### DISCUSSION THREAD

**Discussion by Outerblog** [seerge/g-helper#2610] [General]
i have asus strix scope keyboard, gladius II mouse and delta headphones. it doesnt detect any

**Comment by seerge** [comment]
@Outerblog it's not supposed to detect your [keyboard](https://github.com/seerge/g-helper/issues/2365) and especially headphones (as they don't have any firmware-level settings).

Your mouse is supported tho, make sure you connect it via dongle or by wire (not via BT) 
https://github.com/seerge/g-helper?tab=readme-ov-file#mouse-asus-mouse-and-other-peripherals-support

**Reply by Outerblog** [reply]
so should i just install armory crate again just to control my mouse and keyboard

**Reply by seerge** [reply]
It's up to you. As I have mentioned above - your mouse is supported.

**Reply by Outerblog** [reply]
ive been using the wired mouse but its still not detecting

**Reply by seerge** [reply]
If you believe something is not working as it should, you are free to submit a [new issue](https://github.com/seerge/g-helper/issues/new/choose) explaining your problem in a high detail and attaching all requested logs and accompanying info. Thanks.
